README.md

```diff
@@ -87,7 +87,7 @@ project's technical documentation:
     with support for building and viewing documentation for your framework and
     its dependencies.
   
-## Getting Started with `docc`
+## Getting Started with developing `docc`
```

Comment:
very minor; the rest of the headers use sentence case

```suggestion
## Getting started with developing `docc`
```

Reply:
Good catch, thank you!